CIP-optimized-uplc-serialization/README.md

```diff
@@ -0,0 +1,327 @@
+---
+CIP: "???"
+Title: optimized UPLC serialization  
+Authors: Michele Nuzzi <michele.nuzzi.2014@gamil.com> 
+Discussions-To: TBD
+Comments-Summary: TBD  
+Comments-URI: TBD  
+Status: Draft  
+Type: Standards Track  
+Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>  
+License: CC-BY-4.0  
+License-Code: Apache-2.0  
+Post-History: <dates of postings to Cardano Dev Forum, or link to thread>  
+---
+
+## Abstract
+
+this document describes the parts of the current serialization algorithm that can be improved and provides the specification and documentation needed in order to implement an optimized version of this one.
```

Comment:
You probably want to clean up the formatting, e.g. use capital case at the beginning of sentences.